mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/jobcluster/JobClusterActor.java

```diff
@@ -790,6 +792,10 @@ public void onJobClusterInitialize(JobClusterProto.InitializeJobClusterRequest i
                 cronManager = new CronManager(name, getSelf(), jobClusterMetadata.getJobClusterDefinition().getSLA());
             } catch (Exception e) {
                 logger.warn("Exception initializing cron", e);
+                getSender().tell(new JobClusterManagerProto.CreateJobClusterResponse(
```

Comment:
maybe separate this catch to known error (SchedulerException) and other error (exception) and use different error code + message.